test/minikube/minikube_tls_admin_test.go

```diff
@@ -177,28 +76,27 @@ func TestKubernetesTLS(t *testing.T) {
 
 	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
 
-	helmChartReleaseName, namespaceName := startAdmin(t, &options, 3, namespaceName)
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &options, 3, namespaceName)
 
 	admin0 := fmt.Sprintf("%s-nuodb-0", helmChartReleaseName)
 
-
 	t.Run("verifyKeystore", func(t *testing.T) {
-		content, err := readAll("../../keys/default.certificate")
+		content, err := testlib.ReadAll(filepath.Join(keysLocation, "nuoadmin.cert"))
```

Comment:
is this the new key or the default key?

Reply:
This is the certificate stored in the `NuoAdmin` keystore. 

Not surer what do you mean by `new` in this case context. This is an old test case where we have a verification step for the certificate in the container keystore after TLS is configured and admin is started. There is no rotation here.

Reply:
The test does the following:
- generate a new certificate
- start admin
- verify that the certificate in the admin is not the one that comes with the image by default.

Given that the test is broken and the verification does not work, I am verifying manually which certificate you passed down.

Reply:
Ah, I understand now... To answer your original question:
> is this the new key or the default key?
I pass the new server certificate. So this raises a new question --- Why the test is passing then?
This happens because of the trailing empty line from the `output` produced by:
`output, err := k8s.RunKubectlAndGetOutputE(t, options, "exec", podName, "--", "nuocmd", "show", "certificate", "--keystore", keystore, "--store-password", password)`.

I suggest comparing that the actual keystore content is the same as the expected one and trimming empty lines. Please check my changes.

Reply:
yeah, you are right. The test is broken :) I am down to any suggestion that fixes it :)